cmdlet-style-guide.md

```diff
@@ -4,62 +4,66 @@ This guide outlines the our practices when writing PowerShell cmdlets. It is a
 work in-progress, and if you have suggestions or altering opinions please feel
 free to submit an issue for discussion.
 
-[RFC2119](http://www.ietf.org/rfc/rfc2119.txt)
+[RFC2119][RFC2119]
 
-# Cmdlets which destroy data MUST use `SupportsShouldProcess`
-# Cmdlets MUST call `ShouldProcess` during data destroying situations (e.g. Deleting a GceDisk or GcsBucket).
+## Cmdlets which destroy data MUST use [`SupportsShouldProcess`][SupportsShouldProcess]
+## Cmdlets MUST call [`ShouldProcess`][ShouldProcess] before executing data destroying operations.
+## [`ConfirmImpact`][ConfirmImpact] MAY be left default and SHOULD NOT be set to `High`.
 
-In order to avoid accidentally destroying data (deleting a file, shutting down a
-virtual machine, etc.) users should be prompt to confirm the operation.
+In order to avoid accidentally destroying data (deleting a GcsBucket, deleting a GceDisk, etc.)
```

Comment:
Should we use the full name (GCS Bucket or Google Cloud Storage Bucket) instead of GcsBucket and GceDisk?